spring-cloud-data-shell/src/test/java/org/springframework/cloud/data/shell/AbstractShellIntegrationTest.java

```diff
@@ -36,27 +44,53 @@
 import org.springframework.util.SocketUtils;
 
 /**
- * Base class for all the Shell integration test classes.
+ * Base class for shell integration tests. This class sets up and tears down
+ * the infrastructure required for executing shell tests - in particular, the
+ * {@link AdminApplication} server.
+ * <p>
+ * Extensions of this class may obtain an instance of {@link StreamCommandTemplate}
```

Comment:
Not sure we want to mention this, or if we do, phrase it as an example, as many more are to follow (_e.g._ task(), counter(), etc...)